pyproject.toml

```diff
@@ -36,11 +36,6 @@ virtualenv = "^20.16.2"
 dill = "^0.3.5.1, <0.3.6"
 importlib-metadata = "^4.12.0"
 
-# TODO: review any of these are bigquery-specific or for firebase
-google-cloud-bigquery = "^2.28.1"
-google-cloud-bigquery-storage = "^2.9.1"
-firebase-admin = "^5.1.0"
```

Comment:
this does not break dbt-bigquery support?

Reply:
Nope, here are test results: https://github.com/fal-ai/fal/actions/runs/3901548260